src/game.cpp

```diff
@@ -10078,6 +10159,7 @@ void game::vertical_move( int movez, bool force )
     // Save all monsters that can reach the stairs, remove them from the tracker,
     // then despawn the remaining monsters. Because it's a vertical shift, all
     // monsters are out of the bounds of the map and will despawn.
+    monster stored_mount = monster();
```

Comment:
```suggestion
    monster stored_mount;
```
You don't need to call the default constructor explicitly.